dts/bindings/gpio/silabs,gecko-gpio-port.yaml

```diff
@@ -0,0 +1,19 @@
+description: SiLabs Gecko GPIO port node
+
+compatible: "silabs,gecko-gpio-port"
```

Comment:
What's the plan with the family specific bindings/compatibles?

Reply:
See my comment above https://github.com/zephyrproject-rtos/zephyr/pull/22423#discussion_r374164670

> After this is merged, we should change the other efm32 and efr32 to use this gecko-gpio instead of efm32-gpio or efr32-gpio.

This is at least my plan :)

Reply:
missed that, thanks